chainer/functions/pooling/max_pooling_2d.py

```diff
@@ -177,7 +177,7 @@ def _forward_ideep(self, gy):
 
         n, c, h, w = self._in_shape
         y_h, y_w = gy[0].shape[2:]
-        x, = self.mpool2d.get_retained_inputs()
+        x = self.mpool2d.get_retained_inputs()[0].data
```

Comment:
Why do not use `array`?

Reply:
Because `data` is widely used in current code base. e.g. https://github.com/chainer/chainer/blob/15d475f54fc39587abd7264808c5e4b33782df9e/chainer/functions/activation/sigmoid.py#L47

Reply:
hmm, I see. But `array` is better.